app_dart/cron.yaml

```diff
@@ -22,9 +22,6 @@ cron:
 - description: check for mergeable commits waiting for the tree to go green
   url: /api/check-waiting-pull-requests
   schedule: every 5 minutes
-- description: insert agent health status to bigquery
```

Comment:
We should remove the request handler as well

Reply:
Will do that in following PR that will target app_dart code.